LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2020, Digital Bazaar, Inc.
```

Comment:
Why doesn't this actually include the text "BSD 3-Clause License" as elsewhere.  Evidently github picks up on this somehow and displays contextual help box.

https://github.com/veres-one/did-veres-one/blob/master/LICENSE

Reply:
I think it just specially handles that on the master branch.  I copied it from [edv-client](https://github.com/digitalbazaar/edv-client/blob/master/LICENSE) where it shows up without that extra text.

Reply:
Let's add that license stuff so it's clear and fix up edv-client too.